src/less/variables.less

```diff
@@ -1,394 +1,403 @@
 // Extra small screen / phone
-//** Deprecated `@screen-xs` as of v3.0.1
-@screen-xs: 480px;
-//** Deprecated `@screen-xs-min` as of v3.2.0
-@screen-xs-min: @screen-xs;
-//** Deprecated `@screen-phone` as of v3.0.1
-@screen-phone: @screen-xs-min;
+@screen-xs:                   480px;
+@screen-xs-min:               @screen-xs;
 //== Media queries breakpoints
 // Small screen / tablet
-//** Deprecated `@screen-sm` as of v3.0.1
-@screen-sm: 768px;
-@screen-sm-min: @screen-sm;
+@screen-sm:                   768px;
+@screen-sm-min:               @screen-sm;
 // Medium screen / desktop
-//** Deprecated `@screen-md` as of v3.0.1
-@screen-md: 992px;
-@screen-md-min: @screen-md;
+@screen-md:                   992px;
+@screen-md-min:               @screen-md;
 // Large screen / wide desktop
-//** Deprecated `@screen-lg` as of v3.0.1
-@screen-lg: 1200px;
-@screen-lg-min: @screen-lg;
+@screen-lg:                   1200px;
+@screen-lg-min:               @screen-lg;
 // X-Large screen / wide desktop
-//** Deprecated `@screen-lg` as of v3.0.1
-@screen-xl: 1800px;
-@screen-xl-min: @screen-xl;
-//** Deprecated `@screen-lg-desktop` as of v3.0.1
-@screen-lg-desktop: @screen-lg-min;
+@screen-xl:                   1800px;
+@screen-xl-min:               @screen-xl;
 // So media queries don't overlap when required, provide a maximum
-@screen-xs-max: (@screen-sm-min - 1);
-@screen-sm-max: (@screen-md-min - 1);
-@screen-md-max: (@screen-lg-min - 1);
-@screen-lg-max: (@screen-xl-min - 1);
-@screen-xl-max: 1800px;
+@screen-xs-max:               (@screen-sm-min - 1);
+@screen-sm-max:               (@screen-md-min - 1);
+@screen-md-max:               (@screen-lg-min - 1);
+@screen-lg-max:               (@screen-xl-min - 1);
+@screen-xl-max:               1800px;
+//== Pane
```

Comment:
removed unnecessary comments copied from bootstrap that don't apply to us, and tabbed out the values to be in a column so it's easier to read.